airflow/dags/gtfs_views/reports_gtfs_schedule_index.sql

```diff
@@ -62,9 +62,15 @@ agency_feeds_on_end_date AS (
         , S.itp_id AS calitp_itp_id
         , S.url_number AS calitp_url_number
         , S.agency_name
+        , COALESCE(FI.has_feed_info, FALSE)
```

Comment:
Maybe I'm not reading this right, but it seems like even if there isn't any feed info file, this still coalesces null values into false. So, I'm not sure how this would activate reports for those feeds that don't have a feed info file. Let me know if I'm missing something.

Reply:
@machow clarified that this is expected behavior.